resources/java/functions/LambdaExpression.java

```diff
@@ -0,0 +1,16 @@
+// Functional interface with a single abstract method (SAM)
```

Comment:
Did you add this file in this PR? I think this should not be part of this PR, right?

Reply:
This looks like a file for issue #85, indeed. It seems like this has been removed from this PR, good.